wasp/apps/steps.py

```diff
@@ -78,6 +78,13 @@ def _reset(self):
         self._wake += 24 * 60 * 60
         wasp.system.set_alarm(self._wake, self._reset)
 
+    def touch(self, event):
+        draw = wasp.watch.drawable
+        draw.fill(0x0000,60,112,180,40)
+        watch.accel.steps = 0      
+        
+        self._update()  
+
```

Comment:
I'm not entirely convinced by introducing touch to reset. I suspect acidental touching will be much more common than people who actually need to reset the counter. Additionally I've never seen another device that implements a full reset of the daily step count (since it would stop you reaching your daily step count goal).

There is certainly scope for adding a "trip counter" to the pedometer to be used to track runs or other exercise but I don't think this should be implemented by resetting with the daily step count.

Reply:
Good point, and that's sort of what I was going for. Just being able to use the counter as a trip counter, but maybe having a counter independent of the master counter would be useful.

What if the tap function makes another counter appear and then a second tap removes that second counter?

The second counter then just calculates a difference between when you first tapped and the present count

Reply:
I'm not sure if it "wants" to be part of the step counter app or not. The main reason to wonder is that a trip counter pairs really well with a timer (many people who want a trip counter to measnre N thousand steps also want a timer to tell them how long it took them).

Reply:
So do you think it would be best to make this a completely separate app? Cuz if this is the case, I don't think it would pair particularly well with the existing stopwatch app either

Reply:
Agree. Currently I'm thinking along the lines of an Activity or Sports app that can track both.